service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -235,4 +273,62 @@ private Others createFirstOtherWithoutAName() {
                     .build()
             )).build();
     }
+
+    @Nested
+    class PrepareCaseDetailsTest {
+        private final String[] keys = {
+            "cmoHearingDateList",
+            "recitals",
+            "schedule"};
+
+        private HashMap<String, Object> data; // Tries to use an ImmutableMap unless specified
+        private CaseManagementOrder.CaseManagementOrderBuilder caseManagementOrderBuilder;
+
+        @BeforeEach
+        void setUp() {
+            data = new HashMap<>();
+            caseManagementOrderBuilder = CaseManagementOrder.builder();
+        }
```

Comment:
Do we need those here? We are not doing anything complex why just not inline?

Reply:
removed and inlined